alcs-frontend/src/app/features/notice-of-intent/decision/decision-v2/decision-v2.component.html

```diff
@@ -0,0 +1,242 @@
+        <ng-container *ngIf="decision.isDraft">
+          <button mat-flat-button color="primary" (click)="onEdit(decision)">Edit Draft</button>
+        </ng-container>
+
+        <div
+          *ngIf="!decision.isDraft"
+          matTooltip="{{ disabledCreateBtnTooltip }}"
+          [matTooltipDisabled]="!isPaused && !isDraftExists"
+        >
+          <button
+            mat-stroked-button
+            color="primary"
+            (click)="onRevertToDraft(decision.uuid)"
+            [disabled]="isDraftExists"
+          >
+            Revert to draft
+          </button>
+        </div>
+      </div>
+
+      <h4>Resolution</h4>
+      <section class="decision-section-no-title">
+        <div class="grid-2">
+          <div>
+            <div class="subheading2">Decision Date</div>
+            {{ decision.date | momentFormat }}
+            <app-no-data *ngIf="decision.date === null || decision.date === undefined"></app-no-data>
+          </div>
+          <div>
+            <div class="subheading2">Decision Maker</div>
+            {{ decision.decisionMaker }}
+            <app-no-data *ngIf="decision.decisionMaker === null || decision.decisionMaker === undefined"></app-no-data>
+          </div>
+
+          <div>
+            <div class="subheading2">Decision Outcome</div>
+            {{ decision.outcome.label }} {{ decision.isSubjectToConditions ? '- Subject to Conditions' : '' }}
+          </div>
+
+          <div class="full-width">
+            <div class="subheading2">Decision Description</div>
+            {{ decision.decisionDescription }}
+            <app-no-data
+              *ngIf="decision.decisionDescription === null || decision.decisionDescription === undefined"
+            ></app-no-data>
+          </div>
+
+          <div>
+            <div class="subheading2">Stats Required</div>
+            <app-inline-boolean
+              [selectedValue]="decision.isStatsRequired ?? null"
+              (save)="onStatsRequiredUpdate(decision.uuid, $event)"
+            ></app-inline-boolean>
+          </div>
+
+          <div *ngIf="decision.outcome?.code === 'RESC'">
+            <div class="subheading2">Rescinded Date</div>
+            <ng-container *ngIf="decision.rescindedDate">{{ decision.rescindedDate | momentFormat }}</ng-container>
+            <app-no-data *ngIf="decision.rescindedDate === null || decision.rescindedDate === undefined"></app-no-data>
+          </div>
+
+          <div *ngIf="decision.outcome?.code === 'RESC'">
+            <div class="subheading2">Rescinded Comment</div>
+            {{ decision.rescindedComment }}
+            <app-no-data *ngIf="!decision.rescindedComment"></app-no-data>
+          </div>
+        </div>
+      </section>
+
+      <h4>Documents</h4>
+      <!--      <section class="decision-section">-->
+      <!--        <noi-decision-documents [editable]="false" [loadData]="false" [decision]="decision"></noi-decision-documents>-->
+      <!--      </section>-->
```

Comment:
I assume this will get uncommented later

Reply:
Yes!